ci-operator/step-registry/cucushift/upgrade/check/cucushift-upgrade-check-commands.sh

```diff
@@ -254,7 +254,18 @@ function test_execution() {
 function summarize_test_results() {
     # summarize test results
     echo "Summarizing test results..."
-    [[ -e "${ARTIFACT_DIR}" ]] || exit 0
```

Comment:
Shall we update ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh together ?

Reply:
Done